packages/utils/src/PDFObject.js

```diff
@@ -0,0 +1,128 @@
+/*
+PDFObject by Devon Govett used below.
+The class is part of pdfkit. See https://github.com/foliojs/pdfkit
+LICENSE: MIT. Included in this folder.
+Modifications may have been applied for the purposes of node-signpdf.
+*/
+
+import PDFAbstractReference from './PDFAbstractReference';
+/*
+PDFObject - converts JavaScript types into their corresponding PDF types.
+By Devon Govett
+*/
+
+const pad = (str, length) => (Array(length + 1).join('0') + str).slice(-length);
+
+const escapableRe = /[\n\r\t\b\f()\\]/g;
+const escapable = {
+    '\n': '\\n',
+    '\r': '\\r',
+    '\t': '\\t',
+    '\b': '\\b',
+    '\f': '\\f',
+    '\\': '\\\\',
+    '(': '\\(',
+    ')': '\\)',
+};
+
+// Convert little endian UTF-16 to big endian
+const swapBytes = (buff) => buff.swap16();
+
+export default class PDFObject {
```

Comment:
Does it need to be export default? Just looking at the diff it seems like all the uses do `import PDFObject from ...`. Will it be consistent with the rest of the code to just `export` and not `export default`?